content/troubleshooting/common-issues.md

```diff
@@ -36,3 +36,13 @@ GitHub repositories won't show up in the dropdown while trying to add an app to
 
 ###### Solution
 Try disconnecting the GitHub integration and then connecting it again.
+
+### Changing repository for an application
+
+###### Description
+When migrating from one repository to another, whether from the same provider or not, you may want to preserve the build history, environment variables etc instead of adding the application again, potentially losing valuable information.
```

Comment:
When migrating from one repository to another, whether from the same provider or not, you may want to preserve the build history, environment variables etc. instead of adding the application again, and potentially losing valuable information.